pushtemplates/src/main/java/com/clevertap/pushtemplates/VideoActivity.java

```diff
@@ -127,8 +146,26 @@ public void onClick(View view) {
 					}
 					setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
 					FrameLayout.LayoutParams params = (FrameLayout.LayoutParams) aspectRatioFrameLayout.getLayoutParams();
-					params.width = (int) (350*getApplicationContext().getResources().getDisplayMetrics().density);
-					params.height = (int) ( 210 * getApplicationContext().getResources().getDisplayMetrics().density);
+					float density=getApplicationContext().getResources().getDisplayMetrics().density;
+					if(density>=2.0 && density<2.5) {
```

Comment:
@JideshNair Same comment as above.
Do we not need any code for Density > 1.5 and < 2.0 ?
Also, can you write this in a logical way, density = 1 and then keep increasing. Here and above?

Reply:
fixed